apps/hash-frontend/src/lib/user-and-org.ts

```diff
@@ -86,7 +88,15 @@ export const constructMinimalUser = (params: {
 
   const simpleProperties = simplifyProperties(userEntity.properties);
 
-  const { shortname, displayName, pinnedEntityTypeBaseUrl } = simpleProperties;
+  const {
+    shortname,
+    displayName,
+    pinnedEntityTypeBaseUrl,
+    enabledFeatureFlag,
+  } = simpleProperties;
+
+  const enabledFeatureFlags =
+    (enabledFeatureFlag as FeatureFlag[] | undefined) ?? [];
 
```

Comment:
@CiaranMn @nonparibus I'm wondering whether we should override the enabled feature flags for users that are in the HASH org in the frontend as a temporary way for these things to become available to us in production? Otherwise no user will be able to see this functionality until we have an `/admin` UI, including us.

Reply:
Following the conversation in standup, I've made it so that HASH instance admins have all feature flags enabled by default in the HASH frontend in https://github.com/hashintel/hash/pull/4167/commits/89786cc7d20d44c598e168ea2e5b31df460bcd4f.

One thing I realised is this won't mean that HASH instance admins won't have the feature flags enabled in the browser plugin, is this something we also want to address? (requires fetching the HASH instance and checking its permissions in the browser plugin)

Or are we okay with this not being available until we have the `/admin` UI?